meinberlin/apps/projects/signals.py

```diff
@@ -20,3 +20,5 @@ def post_save_delete(sender, instance, update_fields, **kwargs):
     cache.delete_many(
         ["projects_active", "projects_future", "projects_past", "private_projects"]
     )
+    # TODO: add redis key for earliest next project starts
+    # TODO: add redis key for earliest next project ends
```

Comment:
@goapunk here I will fetch the earliest start and latest end date from phase objects and set them in redis db